civic_sandbox/helpers.py

```diff
@@ -0,0 +1,70 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from .models import BikeParking
+from .serializers import BikeParkingSerializer
+import json 
+from django.contrib.gis.geos import GEOSGeometry, MultiPoint, MultiPolygon, MultiLineString
+from pprint import pprint 
+
+
+def sandbox_view_factory(model_class, serializer_class, multi_geom_class, geom_field, attributes, dates):
+    @api_view(['GET'])
+    def sandbox_function(request, date_filter=dates['default_date_filter'], format=None): 
+        print('\n====================================================================================\n')
```

Comment:
generally `logger.debug('message')` or `if settings.DEBUG: print('message')` is preferred here but I don't ever plan on looking at these logs.  Just thought I'd mention it.